packages/validation/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": { "outDir": "libDev" },
+    "references": [{ "path": "../utils" }]
```

Comment:
I beleive you must to define this manually, but if you specify correct dependecies in `package.json` and run `yarn refs` afterwards it will be generated automatically